test/unit/lib/errors.js

```diff
@@ -30,6 +30,9 @@ test('message check', function(t) {
       case 'VehicleStateError':
         t.regex(new errors[error]('R2D2', {code: 'VS_000'}).message, /R2D2/);
         break;
+      case 'SmartcarErrorV2':
+        t.regex(new errors[error]('R2D2').description, /R2D2/);
```

Comment:
should we add another case here for the V2 error response case? (i.e. `new errors[error]({ object with the new properties})`)

Reply:
yep! added